doc/sphinx/user/methods/Stokes-solver/solver.md

```diff
@@ -0,0 +1,110 @@
+#Solver
+
+The Stokes system we are solving is of the
+form
+
+```{math}
+:label: eq:aligned
+    Mx&=\begin{pmatrix}A & B^T \\
+    B & 0
+    \end{pmatrix}
+    \begin{pmatrix} u\\
+    p
+    \end{pmatrix} =
+    \begin{pmatrix}
+    f \\
+    0
+    \end{pmatrix}
+```
+
+. Here, $A$ is the viscous stress matrix,
+$B^T$ comes from integrating 
+grad on pressure space by parts after
+testing with $v$, and $B$ comes from
+integrating div $u$ by parts after testing
+with $q$.
```

Comment:
This isn't right. When you go from the strong to the weak form, you only integrate the `grad p` term by parts. The `div u` part term is not integrated by parts.